packages/shared/src/dto/workflows/json-schema-dto.ts

```diff
@@ -14,61 +14,61 @@ export type JSONSchemaVersion = string;
 
 export type JSONSchemaDefinition = JSONSchemaDto | boolean;
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-export interface JSONSchemaDto {
-  type?: JSONSchemaTypeName | JSONSchemaTypeName[] | undefined;
-  enum?: JSONSchemaType[] | undefined;
-  const?: JSONSchemaType | undefined;
+export type JSONSchemaDto = Readonly<{
```

Comment:
These changes are for compatibility with `json-schema-to-ts`'s declaration of Json Schema 7 typings.